### PR TITLE
Update Edge `import` support to Edge 16.

### PR DESCRIPTION
**Pull request by jdalton**
Update Edge `import` support to Edge 16 😋

### CODE REVIEW

**Comment by jdalton** [comment]
Related: It looks like `export` [just has `null` data](https://github.com/mdn/browser-compat-data/blame/master/javascript/statements.json#L610-L665). How is this being used by mdn's [`exports` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)?

**Comment by Elchi3** [comment]
Thanks for your first contribution to MDN compat data!

> Related: It looks like `export` [just has `null` data](https://github.com/mdn/browser-compat-data/blame/master/javascript/statements.json#L610-L665). How is this being used by mdn's [`exports` page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export)?

I think there are two compat tables on MDN where `export` appears:
- On the (simple) [`export` keyword page](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/export#Browser_compatibility). In the compat data this is [statements.json#L776-841](https://github.com/mdn/browser-compat-data/blame/master/javascript/statements.json#L776-841)
- On the [page describing the `default` keyword](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/default#Browser_compatibility) (which can be used with `switch` or with `export`) . In the compat data this is [statements.json#L610-L665](https://github.com/mdn/browser-compat-data/blame/master/javascript/statements.json#L610-L665).

So, the one with the null data is describing support for default exports.

Almost all compat data entries have a property called `mdn_url`. It's useful to map an MDN page to the compat data in this repo.

File: javascript/statements.json [hunk 1]
```diff
@@ -1563,11 +1563,7 @@
               "version_added": "61"
             },
             "edge": {
-              "version_added": "15",
```
Comment (teoli2003) [inline javascript/statements.json]: We should keep the old value (and put the new one in front). Something like (indentation incorrect):
            {
              "version_added": "16"
            },
            {
               "version_added": "15",
              "flag": {
                "name": "Experimental JavaScript Features",
                "type": "preference"
              }

**Comment by Elchi3** [comment]
Opened https://github.com/mdn/browser-compat-data/pull/668 that keeps the flag information.

**Pull request closed without merging**